Source/WebCore/page/Page.cpp

```diff
@@ -2152,7 +2152,8 @@ void Page::updateRendering()
         document.evaluateMediaQueriesAndReportChanges();        
     });
 
-    runProcessingStep(RenderingUpdateStep::AdjustVisibility, [&] (auto& document) {
+    // FIXME: This suppression shouldn't be needed.
+    SUPPRESS_UNCOUNTED_LAMBDA_CAPTURE runProcessingStep(RenderingUpdateStep::AdjustVisibility, [&] (auto& document) {
```

Comment:
`NOESCAPE` didn't do the trick?

Reply:
There was a bug in the checker: https://github.com/llvm/llvm-project/pull/125662